i18n/en.json

```diff
@@ -359,6 +359,10 @@
     "id": "api.command.duplicate_trigger.app_error",
     "translation": "This trigger word is already in use. Please choose another word."
   },
+  {
+    "id": "api.command.execute_command.create_post_failed.app_error",
+    "translation": "Command '{{.Trigger}}' failed to post response. Please contact the system admin."
```

Comment:
Thanks @mickmister! 

We're not completely consistent on the use of 'system admin', but propose updating `system admin` to `System Administrator` which is used most often in en.json.

Reply:
included `your System Administrator` change